app/models/spree/variant.rb

```diff
@@ -199,6 +199,10 @@ def amount_in(currency)
       price_in(currency).try(:amount)
     end
 
+    def changed?
+      super || price_changed?
+    end
```

Comment:
Unless I missed something, this is not covered by the specs , could we fix this ?

Reply:
🫢 I will add now

Reply:
Of course, that wasn't as simple as it appeared.
Thanks for the prompt, by adding the spec it became clearer that this wasn't quite the perfect solution, and indeed the `price_changed?` method is not needed and can be cleaned up.